apps/legacy-api/src/common/networks.ts

```diff
@@ -0,0 +1 @@
+export type SupportedNetwork = 'mainnet' | 'testnet'
```

Comment:
I think it would make sense to put this in [NetworkValidationPipe.ts](https://github.com/DeFiCh/jellyfish/pull/1089/files/75245ce621d2cc98ecea38979424ca401bc2b359..2c5cec3130c7328738b3f57e2e54d3880d14ad5f#diff-3053d4c9b6e6b66c60ae4ed501681937046f662c90d30b50ed74832ece2a04e3) since the concerns are tightly related.

Reply:
yup makes sense